src/CG/CG_elem.cpp

```diff
@@ -2989,6 +3058,36 @@ double areaTriangle( array3D const &a, array3D const &b, array3D const &c)
     return 0.5 *norm2(crossProduct(b-a,c-a));
 }
 
+/*!
+ * computes the derivative of the triangle area
+ * with respect to the last given vertex
+ * \param[in] a first vertex coordinates
+ * \param[in] b second vertex coordinates
+ * \param[in] c third vertex coordinates
+ * \return the area derivative with respect to the coordinates of vertex c
+ */
+array3D areaTriangleDerivative( array3D const &a, array3D const &b, array3D const &c)
+{
+    array3D d0_c = {1.0, 0.0, 0.0};
+    array3D d1_c = {0.0, 1.0, 0.0};
+    array3D d2_c = {0.0, 0.0, 1.0};
+
+    array3D vec = crossProduct(b-a,c-a);
+
+    array3D d0_vec = crossProduct(b-a,d0_c-a);
+    array3D d1_vec = crossProduct(b-a,d1_c-a);
+    array3D d2_vec = crossProduct(b-a,d2_c-a);
+
+    vec /= norm2(vec);
```

Comment:
Move this line near the evaluation of vec.

Reply:
I deleted the function. It's not used anymore.